goss.py

```diff
@@ -1,8 +1,11 @@
-#!/usr/bin/env python
```

Comment:
I guess the code is now only compatible with Python >= 3? It's not really an issue but we should state it explicitly since Ansible is still working with Python 2 right?

Reply:
Python 2 hour end of life 01.01.2020. no one should use anymore.